test/Dockerfile-gotest

```diff
@@ -0,0 +1,9 @@
+FROM gcr.io/k8s-ci-on-gke/test-image:0.0
+
+RUN go get -u github.com/golang/lint/golint
+
+ENV REPOSITORY test-infra
+ENV IMPORT_PATH github.com/kubernetes
+ENV TIMEOUT 10
+
+CMD ["go get ./... && golint ./... && go vet ./... && go test ./..."]
```

Comment:
Please move these and others into a bash file and copy that bash file into here.

This will make it easier to do the same thing inside and outside of the container.